src/inputs/labeled-inputs.scss

```diff
@@ -106,15 +106,19 @@
       }
     }
 
-    > .iui-input,
-    > .iui-textarea {
-      padding-right: $iui-xl;
-
-      + .iui-message {
-        position: absolute;
-        top: 50%;
-        right: $iui-sm;
-        transform: translateY(-50%);
+    &.iui-positive,
```

Comment:
Why this change is needed?

Reply:
I was encountering this issue.  A narrow, inline text field is reserving the space for a status icon & encroaching on the text.
![Screen Shot 2021-04-29 at 9 58 39 AM](https://user-images.githubusercontent.com/849817/116562805-577a1700-a8b8-11eb-9c64-5b6436cc2e63.png)

Reply:
If you decide to go with my suggestion, instead of adding status classes you could do the same trick you did in Header component - checking if input is the last element and only then applying padding.

Reply:
The padding is no longer applied based on positive, warning, or negative classes.  Instead we always apply the extra padding, but if the input is the last element (meaning there is no status message provided) then we remove the extra padding.

I am still have a padding issue for non-inline inputs, [see the convo here](https://github.com/iTwin/iTwinUI/pull/51#discussion_r624160174).